src/tests/end-to-end/tests/details-view/automated-tabstops.test.ts

```diff
@@ -17,86 +20,113 @@ describe('Automated TabStops Results', () => {
+                tabStopsSelectors.automatedChecksResultSection,
+            );
+            expect(element).toBeNull();
 
-    test('Detect and display out of order failures', async () => {
-        await openTabStopsPage('tab-stops/out-of-order.html');
+            await verifyTargetPageVisualization(0, 0, 0, 0);
+        },
+        longRunningTabStopsTestTimeout,
+    );
 
-        for (let i = 0; i < 4; i++) {
-            await targetPage.keyPress('Tab');
-            await targetPage.waitForSelectorInShadowRoot(TabStopShadowDomSelectors.svg);
-        }
+    test(
+        'Detect and display out of order failures',
+        async () => {
+            await openTabStopsPage('tab-stops/out-of-order.html');
 
-        await detailsViewPage.waitForSelector(tabStopsSelectors.automatedChecksResultSection);
-        await detailsViewPage.clickSelector(tabStopsSelectors.failedInstancesExpandButton);
+            for (let i = 0; i < 2; i++) {
+                await targetPage.keyPress('Tab');
+                await targetPage.waitForSelectorInShadowRoot(TabStopShadowDomSelectors.svg);
+            }
 
-        const ruleDetails = await detailsViewPage.getSelectorElements(
-            tabStopsSelectors.failedInstancesContent,
-        );
+            await detailsViewPage.waitForSelector(tabStopsSelectors.automatedChecksResultSection);
+            await detailsViewPage.clickSelector(tabStopsSelectors.failedInstancesExpandButton);
 
-        expect(ruleDetails).toHaveLength(2);
-    });
+            const ruleDetails = await detailsViewPage.getSelectorElements(
+                tabStopsSelectors.failedInstancesContent,
+            );
 
-    test('Detect and display unreachable elements failures', async () => {
-        await openTabStopsPage('tab-stops/unreachable.html');
+            expect(ruleDetails).toHaveLength(2);
 
-        for (let i = 0; i < 4; i++) {
-            await targetPage.keyPress('Tab');
-            await targetPage.waitForSelectorInShadowRoot(TabStopShadowDomSelectors.svg);
-        }
+            await verifyTargetPageVisualization(1, 1, 2, 0);
+        },
+        longRunningTabStopsTestTimeout,
+    );
 
-        await detailsViewPage.waitForSelector(tabStopsSelectors.automatedChecksResultSection);
-        await detailsViewPage.clickSelector(tabStopsSelectors.failedInstancesExpandButton);
+    test(
+        'Detect and display unreachable elements failures',
+        async () => {
+            await openTabStopsPage('tab-stops/unreachable.html');
 
-        const ruleDetails = await detailsViewPage.getSelectorElements(
-            tabStopsSelectors.failedInstancesContent,
-        );
+            for (let i = 0; i < 5; i++) {
+                await targetPage.keyPress('Tab');
+                await targetPage.waitForSelectorInShadowRoot(TabStopShadowDomSelectors.svg);
+                // Wait longer since this is a keyboard trap
+                await targetPage.waitForTimeout(500);
```

Comment:
I think it might be a good idea here to actually expose/export the timeout in our source code (the orchestrator I believe) as a const and then use that here: makes it a lot easier to understand the context.